src/include/data_augmentation.h

```diff
@@ -48,22 +49,16 @@ class DataAugmentation {
   //
   // @param out[in]: Vector of OpenCV Matrices
   void SetOut(cv::Mat out);
```

Comment:
Pass all arguments to funtions as references cv::Mat &out

Reply:
Done